packages/core/src/types.ts

```diff
@@ -1960,3 +1989,11 @@ export type StateValueFrom<TMachine extends AnyStateMachine> = Parameters<
 
 export type StateFromMachine<TMachine extends AnyStateMachine> =
   TMachine['initialState'];
+
+export interface PersistedMachineState<TState extends AnyState> {
+  [key: string]: any;
```

Comment:
can we do better here? :p

Reply:
5dc9f4668c6c99085d19f7e578f6480b460dd5ed